css-flexbox/align-items.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+#card-deck {
+  display: flex;
+min-height: 100vh;
+align-items: center;
+justify-content: center;
+}
+.card-wrapper {
+  flex-basis: calc(100% / 3);
+}
```

Comment:
Please fix this indentation 🤢

Reply:
![image](https://user-images.githubusercontent.com/117887912/206562945-6beb1e53-3c71-4273-a63d-9f6e32533ac6.png)